packages/client/hooks/useControlBarCovers.ts

```diff
@@ -40,9 +41,11 @@ export const useCoverable = (
   ref: RefObject<HTMLDivElement>,
   height: number,
   parentRef?: RefObject<HTMLDivElement>,
-  columnsRef?: RefObject<HTMLDivElement>
+  columnsRef?: RefObject<HTMLDivElement>,
+  meetingType?: MeetingTypeEnum
 ) => {
   const updateCoverables = () => {
+    if (meetingType === 'poker') return
```

Comment:
can remove?

Reply:
No, we need both `meetingType` checks here. Without this line, the ref's height will be adjusted and without line 84, `isExpanded` will be false meaning the height of the thread wrapper won't be 100%

Reply:
ahh gotcha LGTM!